### PR TITLE
feat(utils): opacity colors and new color vars

### PR DESCRIPTION
**Pull request by PablitoGS**

### CODE REVIEW

File: src/utils/_opaciy.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// --- Opacity --- //
+
+// alpha parameter for rgba Colors
+// @include bgcolor-opacity($c-black);
+
+$opacity-value: (15, 25, 50, 75, 100) !default;
+
+@mixin bgcolor-opacity ($color) {
```
Comment (kikoruiz) [inline src/utils/_opaciy.scss]: We already have this function defined: https://github.com/SUI-Components/sui-theme/blob/master/src/utils/_colors.scss#L15, does the new one something different?, and, should we follow different solutions to apply transparency to colors?

Comment (kikoruiz) [reply src/utils/_opaciy.scss]: Ok, I see, this only applies to background opacities, nothing more to say 😄

File: src/utils/_opaciy.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// --- Opacity --- //
+
+// alpha parameter for rgba Colors
+// @include bgcolor-opacity($c-black);
+
+$opacity-value: (15, 25, 50, 75, 100) !default;
```
Comment (kikoruiz) [inline src/utils/_opaciy.scss]: Is it validated with UX? Maybe we should move it to settings. Not sure if utils should define variables within their files.

File: src/utils/_opaciy.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// --- Opacity --- //
```
Comment (kikoruiz) [inline src/utils/_opaciy.scss]: Typo in the filename: `src/utils/_opaciy.scss`

File: src/utils/_opaciy.scss (hunk 1 above)
Comment (PablitoGS) [reply src/utils/_opaciy.scss]: yes, I was just going to write

File: src/utils/_opaciy.scss (hunk 2 above)
Comment (PablitoGS) [reply src/utils/_opaciy.scss]: Yes, it's validated, ok you're right, I move it to settings

File: src/utils/_opaciy.scss (hunk 3 above)
Comment (PablitoGS) [reply src/utils/_opaciy.scss]: thanks, fixed!

**Comment by PablitoGS** [comment]
@JoanClaret we need them for color and image panel component.

**Pull request merged**